app/kumactl/cmd/install/testdata/install-control-plane.tproxy-ebpf-experimental-enabled.golden.yaml

```diff
@@ -165,6 +165,7 @@ rules:
       - meshfaultinjections
       - meshhealthchecks
       - meshhttproutes
+      - meshloadbalancings
```

Comment:
Not sure if it sounds good, maybe we should consider renaming the policy to `MeshLoadBalancer`

Reply:
Renamed

Reply:
I think the `Policy` suffix is kind of implied for all resources, so I'd prefer to keep it short. Are you ok with `MeshLoadBalancer`?

Reply:
Hmm I think that's gonna tend to be confusing given that a "load balancer" is typically a piece of infrastructure. Of course `Policy`, `Settings`, whatever is implied but here it would be just to make the name tolerable.

Reply:
Agreed to rename to `MeshLoadBalancingStrategy`